front/src/modules/simulationResult/components/SpaceTimeChart/SpaceTimeChartV2.tsx

```diff
@@ -214,9 +198,10 @@ const SpaceTimeChartV2 = (props: SpaceTimeChartV2Props) => {
       if (selectedTrain.schedule) {
         schedule = selectedTrain.schedule.map((scheduleStep) => {
           if (scheduleStep.arrival) {
+            // arrival is in format PTxxxS (PT3600S) as the number od seconds from the start
             const newScheduleStep = {
               ...scheduleStep,
-              arrival: formatToIsoDate(isoDateToMs(scheduleStep.arrival) + dragOffset * 1000),
+              arrival: scheduleStep.arrival + dragOffset,
```

Comment:
Are you sure you can add the dragOffset like this ? :thinking:

Reply:
Forgot to finish this my bad, thank you !